hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -321,15 +321,16 @@ public int size() {
     }
 
     @Override
-    public void doEviction(boolean withoutMaxSizeCheck) {
+    public boolean doEviction(boolean withoutMaxSizeCheck) {
         checkAvailable();
 
         if (evictionDisabled) {
-            return;
+            return false;
         }
 
         EvictionChecker evictionChecker = withoutMaxSizeCheck ? null : this.evictionChecker;
         evictionStrategy.evict(records, evictionPolicyEvaluator, evictionChecker, this);
+        return true;
```

Comment:
Just to double-check - the idea is to return true if eviction was performed, but not if eviction was performed **and** an entry was evicted (that is, return the result of `evictionStrategy.evict()`), correct?

Reply:
your understanding is true.